src/index.js

```diff
@@ -1 +1,64 @@
 import './style.css';
+
+const scores = document.querySelector('.scores > ul');
+const getAPIData = () => {
+  async function getFetch() {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9I754jQkwVXa66yLA2rE/scores',
+    );
+    if (!response.ok) {
+      throw new Error(`API error! status: ${response.status}`);
+    }
+    const data = await response.json();
+    return data;
+  }
```

Comment:
- [ ] In this milestone it is required to use ES6 syntax. Kindly refactor this function to an arrow function. :pray: Your app works great though! :clap: :clap: :clap:

Reply:
Hi @henatan99, thank you for your reviewing my project. I will change these functions to arrow functions.
Thank you for your suggestion. I will create new files to simplify my app and make it more readble.